pkg/printer/formatter.go

```diff
@@ -38,3 +40,13 @@ func uint64Grouping(n uint64) string {
 		}
 	}
 }
+
+// formatDurationNS formats a duration expressed in nanosecond to a human
+// readable duration. For example, 100_000_000_000 is formatted as 1m40s.
+func formatDurationNS(ns uint64) string {
```

Comment:
Could we turn ns into a Duration and use normal %d printer?

Reply:
I'm not sure I understand your comment as this is basically what this function does (see the return statement). The only different is that it accounts for cases where we have a uint64 that would overflow an int64 (unlikely to happen in practice but probably best to handle it) which is the base time of a time.Duration.